config/development.yaml

```diff
@@ -1 +1 @@
-development-cass.yaml
\ No newline at end of file
+development-postgres12.yaml
```

Comment:
I think this should be consistent with `make start`. It can be either sqlite or cass-es I guess.